test/package.json

```diff
@@ -16,7 +16,6 @@
     "bedrock-jsonld-document-loader": "^1.0.1",
     "bedrock-kms": "file:..",
     "bedrock-mongodb": "^8.2.0",
-    "bedrock-ledger-context": "^18.0.0",
```

Comment:
`bedrock-veres-one-context` has a peer dependency for `bedrock-ledger-context`

https://github.com/digitalbazaar/bedrock-veres-one-context/blob/master/package.json#L30

Was this removed for a particular reason?